core/primitives/big_int.hpp

```diff
@@ -31,6 +31,11 @@ namespace fc::primitives {
 namespace fc {
   using primitives::bigdiv;
   using primitives::bigmod;
+  using primitives::BigInt;
+
+  inline auto bitlen(const BigInt &x) {
```

Comment:
Why do we want it in big_int.hpp? Why not math.hpp?

Reply:
because it's BigInt methid